### DISCUSSION TITLE
I have 40 json files, anybody tell me how to combine them to 1 json file like coco2017?

### DISCUSSION THREAD

**Discussion by Shaobo-Zhao** [wkentaro/labelme#1394] [Q&A / Help]
I have 40 json files, anybody tell me how to combine them to 1 json file like coco2017?
please please

**Comment by wkentaro** [comment]
@Shaobo-Zhao Have you looked at [`labelme2coco.py`](https://github.com/labelmeai/labelme/blob/main/examples/instance_segmentation/labelme2coco.py)?

Here is the README: https://github.com/labelmeai/labelme/tree/main/examples/instance_segmentation#convert-to-coco-format-dataset

**Reply by Shaobo-Zhao** [reply]
Thank you for your reply. I m sorry for seeing your reply so late ，I combined my json files into 1 json file successfully.(useing label2coco.py)

**Comment by Shaobo-Zhao** [comment]
> @Shaobo-Zhao Have you looked at [`labelme2coco.py`](https://github.com/labelmeai/labelme/blob/main/examples/instance_segmentation/labelme2coco.py)?
> 
> Here is the README: https://github.com/labelmeai/labelme/tree/main/examples/instance_segmentation#convert-to-coco-format-dataset
